src/infer_context.cc

```diff
@@ -103,6 +103,12 @@ InferContext::CompleteOngoingSequence(uint32_t seq_stat_index)
   }
 }
 
+void
+InferContext::ApplyDatasetOffset(size_t dataset_offset)
+{
+  data_step_id_ += dataset_offset;
+}
+
```

Comment:
Ultimately, the InferContext inside of the RequestRateWorker needs to know what the dataset offset from [here](https://github.com/triton-inference-server/perf_analyzer/pull/366/files#r2058824073) is in order to skip it during benchmarking.